docs/src/main/paradox/server/grpc-web.md

```diff
@@ -37,9 +37,9 @@ gRPC-Web endpoint with basic CORS infrastructure in place. To use CORS,
 you will need to add the pekko-http-cors dependency to your project:
 
 @@dependency[sbt,Maven,Gradle] {
-  group="ch.megard"
+  group="org.apache.pekko"
   artifact="pekko-http-cors_2.12"
-  version="0.0.0-SNAPSHOT" // Update when proper release of pekko-http-cors is made
+  version="0.0.0-4456-9a4c0fc7-SNAPSHOT" // Update when proper release of pekko-http-cors is made
```

Comment:
shouldn't this be derived by the build (sbt-dynver)?

Reply:
Nice find, gimme a sec

Reply:
So I just realized that the `@@dependency` directive in this project is broken, as in it doesn't even render correctly. Looking into it

Reply:
Just pushed and I can confirm its rendering correctly, i.e.

![image](https://github.com/apache/incubator-pekko-grpc/assets/2337269/691448ba-f57e-41e2-b45f-002f7a0736a0)

![image](https://github.com/apache/incubator-pekko-grpc/assets/2337269/250cd862-a64a-49bc-b57a-4bd5d79ac029)

![image](https://github.com/apache/incubator-pekko-grpc/assets/2337269/80124481-5477-4a32-99e3-9369bae66fb4)